custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
[via ChatGPT review bot] It is a small change but it would be helpful if you use variable names that define what the color is rather than what it is used for. This would improve the readability of your code, especially when your color scheme becomes larger.

You may refer to [Sass Guidelines](https://sass-guidelin.es/) for more tips on using SASS.

```scss
$color-light: #F8EFF4;
$color-dark: #10091d;
$color-highlight: #F8EFF4;
$color-light-text: #a7baca;
$color-lowlight-text: #535d65;
$color-dark-text: #10091d;
```